ChelseaGreen-chelseagreen/hw-2/hw2_rubymonk.rb

```diff
@@ -0,0 +1,52 @@
+#Length of strings in an array. 
+def length_finder(input_array)
+  input_array.map do |string_length| string_length.length
+  end 
```

Comment:
While this one-liner is valid code, it's best practice to use `do/end` block syntax as an actual block with `string_length.length` as a new line.

Still, this is well written and concise code. :+1: